### ISSUE TITLE
[Safari] Missing focus on upselling products on PDP

### ISSUE THREAD

**Issue by KateChuen** [SAP/spartacus#6575]
**Environment Details**
Spartacus: epic/accessibility branch
Browser: Chrome & Safari
Device: mac
Happens on: http://dev-com-3.accdemo.b2c.ydev.hybris.com:4200/electronics-spa/en/USD

**Steps to Reproduce**
- Go to Product Details Page of a product that has cross-selling/upselling products, for eg: 1981415
- Tab to the cross-selling/upselling products.

**Actual Result**
Focus missing.

<img width="1244" alt="Screen Shot 2020-02-14 at 1 21 14 PM" src="https://user-images.githubusercontent.com/45409515/74557572-68966c80-4f2e-11ea-8414-0710f9eed5b5.png">

**Comment by developpeurweb** [comment]
Fixed for Chrome and Forefox

![Screen Shot 2020-02-28 at 4 37 02 PM](https://user-images.githubusercontent.com/15113219/75589407-acb66080-5a48-11ea-8407-7a80478ec801.png)
![Screen Shot 2020-02-28 at 4 37 31 PM](https://user-images.githubusercontent.com/15113219/75589408-ad4ef700-5a48-11ea-90c0-47af12d1f037.png)

**Comment by KateChuen** [comment]
Tested on dev10.
It's true it's fixed on chrome and firefox. 

@developpeurweb But will it be fixed for safari too?

<img width="1177" alt="Screen Shot 2020-03-26 at 10 18 33 AM" src="https://user-images.githubusercontent.com/45409515/77657277-771f6d00-6f4b-11ea-80da-7f1d9f2f2048.png">

**Comment by developpeurweb** [comment]
Now fixed for Safari as well.

<img width="1396" alt="Screen Shot 2020-04-22 at 2 09 44 PM" src="https://user-images.githubusercontent.com/15113219/80017690-f9e00d00-84a2-11ea-9867-e3b91f6ab5cb.png">

**Comment by KateChuen** [comment]
Tested on dev3. 
✅Now we have a visual focus on safari too. 
<img width="1194" alt="Screen Shot 2020-04-28 at 10 37 48 AM" src="https://user-images.githubusercontent.com/45409515/80500752-8fb0e780-893c-11ea-83b1-3bfc046ac113.png">
